mule-extensions-api/src/main/java/org/mule/runtime/extension/api/ExtensionConstants.java

```diff
@@ -42,6 +42,23 @@
   public static final String TARGET_PARAMETER_DISPLAY_NAME = "Target Variable";
 
   /**
+   * The name of a synthetic parameter that's automatically added to all non void operations. The meaning of it is requesting the
+   * runtime to configure whether the entire operation's output {@link Message} or just its payload will be stored in the flowVar pointed by the target parameter.
+   */
+  public static final String TARGET_OUTPUT_PARAMETER_NAME = "targetOutput";
```

Comment:
Isn't `targetType` more explicit, given that this attribute is defining the type of value set using target flowVar? Or is the feeling that given it takes a enum value, together with the possible values, which are types, makes this more explicit/self-explanatory?